rviz_common/src/rviz_common/frame_manager.hpp

```diff
@@ -31,18 +31,18 @@
 #ifndef RVIZ_COMMON__FRAME_MANAGER_HPP_
 #define RVIZ_COMMON__FRAME_MANAGER_HPP_
 
-#include <geometry_msgs/msg/pose.hpp>
-#include <rclcpp/time.hpp>
+#include <map>
+#include <memory>
+#include <mutex>
+#include <string>
 
 #include <OgreVector3.h>
 #include <OgreQuaternion.h>
 
-#include <QObject>
+#include <QObject> // NOLINT: cpplint is unable to handle the include order here
 
-#include <map>
-#include <memory>
-#include <mutex>
-#include <string>
+#include <geometry_msgs/msg/pose.hpp> // NOLINT: cpplint is unable to handle the include order here
+#include <rclcpp/time.hpp> // NOLINT: cpplint is unable to handle the include order here
```

Comment:
This shouldn't pass uncrustify (or maybe it's cpplint and that's why it doesn't show up), but you need two spaces between code and comment.

Also you can just use `"` here instead of `<` to shut cpplint up.

Reply:
Same in other files.

By the way I didn't do this in the Qt headers case because that goes against all of the Qt examples so I was constantly changing the include style and there was a lot of existing instances. But for our libraries we're including, this style is probably better.

A harder, but more awesome 😄, solution would be to extend our tool `ament_cpplint` (and it's embedded copy of cpplint.py) to not be so picky in these cases.

Reply:
Please check the two spaces before trailing comments in other places where `NOLINT` is used too.

Reply:
For now I will go with `"` for ros includes.